cl/recap/tasks.py

```diff
@@ -1660,13 +1667,10 @@ def process_recap_email(
             send_alert_and_webhook.delay(
                 docket.pk, start_time, recap_email_recipients
             )
-    return {
-        "docket_pk": docket.pk,
-        "content_updated": bool(rds_created or content_updated),
-    }
+    return [rd.pk for rd in rds_to_download]
 
 
 def do_recap_document_fetch(epq: EmailProcessingQueue, user: User) -> None:
     return chain(
-        process_recap_email.si(epq.pk, user.pk),
+        process_recap_email.s(epq.pk, user.pk), extract_recap_pdf.s()
```

Comment:
I'm surprised you switched from `si()` to `s()` here. I thought `si()` was for later elements in a chain, to tell them not to use the return value of the prior task as their first input?

Reply:
Yeah, you're right. process_recap_email.si() doesn't need to change to s() because extract_recap_pdf it's the one that need to accept the param from the previous task. 

Changed, thanks.